### DISCUSSION TITLE
zksync bridge

### DISCUSSION THREAD

**Discussion by TaiwoEniola** [Cyfrin/foundry-full-course-cu#2278] [General]
whenever I select "zksync sepolia testnet" on the zksync bridge it keeps showing "logging in to Metamask" for a long time and it doesn't move pass there. i have not been able to convert my sepolia ethereum  to zysync sepolia testnet

**Comment by EngrPips** [comment]
Please try another browser.

**Reply by TaiwoEniola** [reply]
i have still the same thing

**Reply by EngrPips** [reply]
Interesting, Honestly, I have run into this same issue before, and clearing my browser history solved it for me. I told you to do the same, but you said it didn't solve your problem. I think you should try another device and see if the issue persists.

**Comment by TaiwoEniola** [comment]
Okay I will try thanks

**Reply by EngrPips** [reply]
You are welcome, I am looking forward to how that will play out.

**Comment by TaiwoEniola** [comment]
I figured it out, I used my Metamask browser and it worked

**Reply by EngrPips** [reply]
Awesome, Finally. I am glad you figured this out.
